plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/popup/CodeWhispererPopupManager.kt

```diff
@@ -484,6 +501,20 @@ class CodeWhispererPopupManager {
             window?.addComponentListener(windowListener)
             Disposer.register(sessionContext) { window?.removeComponentListener(windowListener) }
         }
+
+        val suggestionHoverEnterListener: EditorMouseMotionListener = object : EditorMouseMotionListener {
+            override fun mouseMoved(e: EditorMouseEvent) {
+                if (e.inlay != null) {
+                    showPopup(sessionContext, force = true)
+                } else {
+                    sessionContext.project.messageBus.syncPublisher(
+                        CodeWhispererService.CODEWHISPERER_INTELLISENSE_POPUP_ON_HOVER,
+                    ).onEnter()
```

Comment:
if we already have the `sessionContext`, why do we need the message bus?

Reply:
don't need it here